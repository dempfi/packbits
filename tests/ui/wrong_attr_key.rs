use packbits as _;

#[packbits::pack(b = 1)]
struct Bad {
  a: u8,
}

fn main() {}
