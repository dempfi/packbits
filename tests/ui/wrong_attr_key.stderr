error: expected `bytes = <int>`
 --> tests/ui/wrong_attr_key.rs:3:18
  |
3 | #[packbits::pack(b = 1)]
  |                  ^
